polaris-react/src/components/Layout/components/AnnotatedSection/AnnotatedSection.tsx

```diff
@@ -19,15 +20,33 @@ export function AnnotatedSection({
   description,
   id,
 }: AnnotatedSectionProps) {
+  const {polarisSummerEditions2023} = useFeatures();
   const descriptionMarkup =
-    typeof description === 'string' ? <p>{description}</p> : description;
+    // eslint-disable-next-line no-nested-ternary
+    typeof description === 'string' ? (
+      polarisSummerEditions2023 ? (
+        <Text as="p" variant="bodyMd">
+          {description}
+        </Text>
+      ) : (
+        <p>{description}</p>
+      )
+    ) : (
+      description
+    );
```

Comment:
Yeah, originally had this abstracted another level with more variables but then figured leaving it this way would make less work for clean up later since its temp 😅